app/src/componentLibrary/fieldText/fieldText.scss

```diff
@@ -210,6 +210,7 @@
       @include asterisk();
       right: -7px;
       content: '*';
+      color: $COLOR--red-focused;
```

Comment:
Should it be in red?

Reply:
As I see, it should be in red.
https://www.figma.com/file/gjYQPbeyf4YsH3wZiVKoaj/%F0%9F%9B%A0-RP-DS-6?type=design&node-id=5158-44324&t=JDqzkhmRNRDSB8Y2-0